ssl_study/simclrv1/pretext/utils/utils.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import tensorflow_similarity as tfsim
+
+# Sanity Check
+class ShowAugmentedBatch:
+    """
+    Script to show image batch in dataset
+    Example usage:
+        show_augmented_batch = ShowAugmentedBatch(config)
+        display_imgs = next(inclassloader.as_numpy_iterator())
+        max_pixel = np.max([display_imgs[0].max(), display_imgs[1].max()])
+        min_pixel = np.min([display_imgs[0].min(), display_imgs[1].min()])
+        show_augmented_batch.show_augmented_batch(display_imgs, max_pixel, min_pixel)
+    """
+
+    def __init__(self, args):
+        self.args = args
+
+    def show_augmented_batch(self, display_imgs, max_pixel, min_pixel):
+        tfsim.visualization.visualize_views(
```

Comment:
What's the interface for this visualization? Will it dump an image in the directory?

Reply:
Also, seeing the visualization, have to show rgb image rather than just blue channel from augmented image. The image shown is actually two view(2 augmented image of same example) stacked together parallely. Though, it can't be well seen because of the blue color.